src/app/helptext/task-calendar/replication/replication.ts

```diff
@@ -2,234 +2,243 @@ import { T } from '../../../translate-marker';
+ replication.'),
 
     target_dataset_placeholder: T('Target Dataset'),
-    target_dataset_tooltip: T('Choose a dataset on the destination\
- system where snapshots are stored. Click the <b>Folder icon</b> to see\
- all datasets on the destination system and click on a dataset to set it\
- as the target. An SSH connection must be selected for the browser to\
- display datasets from a remote system.'),
+    target_dataset_tooltip: T('Choose a dataset on the destination \
+ system where snapshots are stored. Click the \
+ <i class="material-icons">folder</i> to see all datasets on the \
+ destination system. Click a dataset to set it as the target.'),
 
     recursive_placeholder: T('Recursive'),
-    recursive_tooltip: T('Replicate all child dataset snapshots. Set to\
- make <b>Exclude Child Datasets</b> visible.'),
+    recursive_tooltip: T('Replicate all child dataset snapshots. When \
+ set, <b>Exclude Child Datasets</b> becomes available.'),
 
     exclude_placeholder: T('Exclude Child Datasets'),
-    exclude_tooltip: T('Exclude specific child dataset snapshots from\
- the replication. Use with <b>Recursive</b> snapshots. List child\
- dataset names to exclude. Example: <i>pool1/dataset1/child1</i>. A\
- recursive replication of <i>pool1/dataset1</i> snapshots includes all\
+    exclude_tooltip: T('Exclude specific child dataset snapshots from \
+ the replication. Use with <b>Recursive</b> snapshots. List child \
+ dataset names to exclude. Example: <i>pool1/dataset1/child1</i>. A \
+ recursive replication of <i>pool1/dataset1</i> snapshots includes all \
  child dataset snapshots except <i>child1</i>.'),
 
     properties_placeholder: T('Properties'),
-    properties_tooltip: T('Send dataset properties along with snapshots.'),
+    properties_tooltip: T('Include dataset properties with the replicated \
+ snapshots.'),
 
     periodic_snapshot_tasks_placeholder: T('Periodic Snapshot Tasks'),
-    periodic_snapshot_tasks_tooltip: T('Snapshot schedule for this\
- replication task. Choose from configured\
- <a href="--docurl--/tasks.html#periodic-snapshot-tasks"\
- target="_blank">Periodic Snapshot Tasks</a>. This replication task must\
- have the same <b>Recursive</b> and <b>Exclude Child Datasets</b> values\
- as the chosen periodic snapshot task. Selecting a periodic snapshot\
- schedule hides the <b>Schedule</b> field.'),
+    periodic_snapshot_tasks_tooltip: T('Snapshot schedule for this \
+ replication task. Choose from configured \
+ <a href="--docurl--/tasks.html#periodic-snapshot-tasks" \
+ target="_blank">Periodic Snapshot Tasks</a>. This replication task must \
+ have the same <b>Recursive</b> and <b>Exclude Child Datasets</b> values \
+ as the chosen periodic snapshot task. Selecting a periodic snapshot \
+ schedule removes the <b>Schedule</b> field.'),
 
     naming_schema_placeholder: T('Naming Schema'),
-    naming_schema_tooltip: T(''),
+    naming_schema_tooltip: T('Custom snapshots to be replicated. Enter \
```

Comment:
This is not the actual snapshots, but a naming pattern that matches the snapshots. We did this more clearly on the other helptext: "Pattern that matches the names of snapshots to be replicated." (or something like that).